src/sagemaker/lineage/query.py

```diff
@@ -259,23 +325,63 @@ def _get_app(self, elements):
                                 "label": "data(label)",
                                 "color": "gray",
                                 "text-halign": "left",
-                                "text-margin-y": "3px",
-                                "text-margin-x": "-2px",
-                                "font-size": "3%",
-                                "width": "1%",
-                                "curve-style": "taxi",
+                                "text-margin-y": "2.5",
+                                "font-size": "3",
+                                "width": "1",
+                                "curve-style": "bezier",
+                                "control-point-step-size": "15",
                                 "target-arrow-color": "gray",
                                 "target-arrow-shape": "triangle",
                                 "line-color": "gray",
                                 "arrow-scale": "0.5",
+                                "font-family": "verdana",
                             },
                         },
-                    ],
+                        {"selector": ".startarn", "style": {"shape": "star"}},
+                        {"selector": ".select", "style": {"border-opacity": "0.7"}},
+                    ]
+                    + [self._create_entity_selector(k, v) for k, v in self.entity_color.items()],
                     responsive=True,
-                )
+                ),
+                self.html.Div(
+                    style={
+                        "width": "0.5%",
+                        "display": "inline-block",
+                        "font-size": "1vw",
+                        "font-family": "verdana",
+                        "vertical-align": "top",
+                    },
+                ),
+                # legend section
+                self.html.Div(
+                    [self._create_legend_component(k, v) for k, v in self.entity_color.items()]
```

Comment:
Won't this result in the string `TrialComponent` being shown instead of ` Trial Component`? Could either string split on capitalization or just include the title in the config as well as the selector id

Reply:
resolved